scripts/apps/monitoring/directives/MonitoringGroup.js

```diff
@@ -435,11 +431,17 @@ export function MonitoringGroup(cards, api, authoringWorkspace, $timeout, superd
                         scope.items = search.updateItems(items, scope.items);
                     }
                 })
-               .finally(() => {
-                   if (originalQuery) {
-                       criteria.source.query = originalQuery;
-                   }
-               });
+                .finally(() => {
+                    if (originalQuery) {
+                        criteria.source.query = originalQuery;
+                    }
+
+                    // finally, if it's a forced refresh then scroll to Top
+                    if (data && data.force) {
+                        scope.scrollTop = 0;
```

Comment:
Seems very hacky to save scrollTop positions on scope. Why not just compute them on the fly where they are needed?